runtime/experimental/Makefile

```diff
@@ -0,0 +1,73 @@
+SHELL := /bin/bash
+
+compiler:
+	docker build -f ${PWD}/compiler.Dockerfile -t bref/runtime/compiler:latest .
+
+shell/compiler:
+	docker run --interactive --tty -v ${PWD}:/host --entrypoint /bin/bash -e "TERM=xterm-256color" bref/runtime/compiler:latest
```

Comment:
I think we can get rid of those "shell/" commands to simplify the Makefile, maybe we could document the `docker run` commands in the readme instead so that they can still be used?

Reply:
Certainly. They were simply convenient for developing the build system itself.

Reply:
Done in commit https://github.com/mnapoli/bref/pull/132/commits/cafe64b1cfc09acf35557e0ddba7838fade54a28